package.json

```diff
@@ -124,5 +124,6 @@
     "rimraf": "5.0.10",
     "ts-node": "10.9.2",
     "typescript": "5.4.5"
-  }
+  },
+  "packageManager": "yarn@1.22.21+sha1.1959a18351b811cdeedbd484a8f86c3cc3bbaf72"
```

Comment:
Heyo @cathysarisky do you know where this has snuck in from? Can we remove it for now?

If it's something super useful for a common tool we can discuss and add separately :)

Reply:
I apologize. That snuck past me.  Will fix!